extensions/amp-access/0.1/amp-access.js

```diff
@@ -350,6 +352,16 @@ export class AccessService {
             credentials: 'include',
             requireAmpResponseSourceOrigin: true
           }));
+    }).catch(error => {
```

Comment:
No prob. We already have a primitive for exponential backoff in place. But I would not retry more than once anyway.